cmd/manager/main_test.go

```diff
@@ -0,0 +1,37 @@
+package main
+
+import (
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+// TestCheckIfRequiredFilesExist tests if checkIfRequiredFilesExist function is throwing appropriate error when some
+// of the files required by WMCO are missing
+func TestCheckIfRequiredFilesExist(t *testing.T) {
+	var missingFiles = []string{
+		"/payload/hybrid-overlay.exe",
+		"/payload/wmcb.exe",
+	}
+	// Saving osStat function and restoring at the end
+	oldOsStat := osStat
```

Comment:
Does this need to be the global variable? You could do the same with `os.Stat` or am I missing something?

Please name it `originalOsStat` or `orgOsStat`

Reply:
This was done so that I can mock it in TestCheckIfRequiredFilesExist in main_test.go. Check [this](TestCheckIfRequiredFilesExist). Let me know if you can suggest any other way to mock os.Stat